kythe/java/com/google/devtools/kythe/platform/shared/KytheInlineMetadataLoader.java

```diff
@@ -0,0 +1,100 @@
+package com.google.devtools.kythe.platform.shared;
+
+import static java.nio.charset.StandardCharsets.UTF_8;
+
+import com.google.common.base.Splitter;
+import com.google.common.base.Utf8;
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.Iterables;
+import com.google.common.flogger.FluentLogger;
+import com.google.common.io.BaseEncoding;
+import com.google.devtools.kythe.analyzers.base.EdgeKind;
+import com.google.protobuf.InvalidProtocolBufferException;
```

Comment:
Please re-order these imports to make google-java-format happy:
```
 import com.google.common.flogger.FluentLogger;
 import com.google.common.io.BaseEncoding;
 import com.google.devtools.kythe.analyzers.base.EdgeKind;
-import com.google.protobuf.InvalidProtocolBufferException;
 import com.google.devtools.kythe.proto.Metadata.GeneratedCodeInfo;
 import com.google.devtools.kythe.proto.Metadata.MappingRule;
+import com.google.protobuf.InvalidProtocolBufferException;
 ```

Reply:
Done.